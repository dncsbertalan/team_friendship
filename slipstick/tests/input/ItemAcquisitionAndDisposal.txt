load testmap.txt
move Student_1 Room_1
pick_up_item FFP2_2
drop_item FFP2_2
exit