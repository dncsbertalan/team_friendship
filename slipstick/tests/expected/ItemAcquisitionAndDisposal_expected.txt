Successfully loaded testmap.txt
Moving Student_1 from MainHall to Room_1
Student_1 picked up FFP2_2
Student_1 dropped FFP2_2
